cmd/peggo/options.go

```diff
@@ -263,12 +265,26 @@ func initRelayerOptions(
 		Value:  false,
 	})
 
+	*relayValsetOffsetDur = cmd.String(cli.StringOpt{
+		Name:   "relay_valset_offset_dur",
+		Desc:   "Sets a duration to delay relaying Valset",
```

Comment:
@mankenavenkatesh please provide a more descriptive description here as to what this delay actually does. Also why is the default 5 min?

Reply:
Default is set to 5 min so that 
1. The validators who set `PEGGO_RELAY_BATCH_OFFSET_DUR`  value less than 5m will be prioritised to send the batch transaction and avoid collision. 
2. Incase no validators set `PEGGO_RELAY_BATCH_OFFSET_DUR` less than 5m, validators with default value of 5m will be prioritised ensuring batch is sent in 5m upon creation and avoid delay.